### ISSUE TITLE
Missing KMLs for topsStack in ISCE 2

### ISSUE THREAD

**Issue by bryanvriel** [forrestfwilliams/burst2safe#72]
Thank you @forrestfwilliams for this project! The burst SAFE files work perfectly well with `topsApp`. I'm now trying to get them to work with `topsStack` using `stackSentinel.py`, but it seems like `topsStack.Stack` is looking for a `map-overlay.kml` file that doesn't exist in the burst SAFE files. For example:

```
stackSentinel.py -s safe -o orbits -a aux -W offset -b '59.919 60.301 -140.829 -140.169' -d nasadem_wgs84.dem
Using default ISCE Path: /Users/briel/apps/isce2/packages/isce
Number of SAFE files found: 4
...
  File "/Users/briel/src/isce2/contrib/stack/topsStack/stackSentinel.py", line 312, in get_dates
    pnts = safeObj.getkmlQUAD(safe)
  File "/Users/briel/src/isce2/contrib/stack/topsStack/Stack.py", line 1765, in getkmlQUAD
    kmlFile = open( file, 'r' ).read(-1)
FileNotFoundError: [Errno 2] No such file or directory: 'safe/S1A_IW_SLC__1SSV_20190410T025508_20190410T025513_026722_030014_75EA.SAFE/preview/map-overlay.kml'
```

Are you currently planning on generating something like an empty KML with the burst boundaries, or do you have another strategy for making `burst2stack` compatible with `topsStack`?

**Comment by forrestfwilliams** [comment]
Hey @bryanvriel do you want to give the version of the tool in PR #73 a try? I think it should fix your issue.

**Comment by forrestfwilliams** [comment]
Hey @bryanvriel closing this issue as it should be fixed as of `v1.1.0`.
